packages/@aws-cdk/aws-elasticloadbalancingv2-actions/.eslintrc.js

```diff
@@ -0,0 +1,2 @@
+const baseConfig = require('../../../tools/cdk-build-tools/config/eslintrc');
```

Comment:
Heads up on incoming PRs - https://github.com/aws/aws-cdk/pull/7880 and https://github.com/aws/aws-cdk/pull/6869